extensions/dicom-seg/src/id.js

```diff
@@ -0,0 +1,3 @@
+const id = 'seg';
+
+export default id;
```

Comment:
Probably just specify this in `index.js`?
Maybe: `com.ohif.dicom-segmentation` for lower chance of collision?

Reply:
This id file is being imporated+used in OHIFDicomSegSopClassHandler, as the value for the plugin property of a segDisplaySet. https://github.com/OHIF/Viewers/blob/b18d1c3b52884c33d443fc33ecb9ecae5758c9d9/extensions/dicom-seg/src/OHIFDicomSegSopClassHandler.js#L33

Reply:
Hmm.. I guess I don't understand the implications. Do they have to be coupled? Do we expect the created display set to have a plugin id that matches a registered extension's id? How are we keying off of this?

I know we use it to match up vtk/cornerstone viewports -- I'm not sure how it impacts seg displaysets.

Reply:
Right now I don't think this is actually keying off anything. We can remove it.
During implementation I thought it might be useful to know which extension made a `displaySet`. Maybe this is still useful to know for debugging purposes/ possibile collisions in the future, but its not being used right now. I don't mind removing it from the `SOPClassHandler`.

Reply:
@dannyrb pending?